docs/docs/config-reference.md

```diff
@@ -349,6 +349,28 @@ This setting is useful when the "Required approving reviews" GitHub Branch Prote
 
 See the "[Automated dependency updates with Dependabot](recipes.md#automated-dependency-updates-with-dependabot)" recipe for an example of this feature in action.
 
+## configuration labels
+
+Some configuration options set in `.kodiak.toml` can be overriden with pull request labels.
+
+Please [open an issue on GitHub](https://github.com/chdsbd/kodiak/issues/new/choose) if you would like support for more options.
+
+### `merge.method`
+
+The [`merge.method`](config-reference.md#mergemethod) option can be overriden with a label, allowing you to merge a pull request with a `merge.method` that differs from your `.kodiak.toml` setting.
+
+#### Example
+
+Label your pull request with the `kodiak: merge.method = 'rebase'` label to set `merge.method` to "rebase" for your pull request.
+
+Here's a table of all the options:
+
+| merge method | label                             |
+| ------------ | --------------------------------- |
+| merge        | `kodiak: merge.method = "merge"`  |
+| squash       | `kodiak: merge.method = "squash"` |
+| rebase       | `kodiak: merge.method = "rebase"` |
```

Comment:
the paragraph above mentions `kodiak: merge.method = 'rebase'` but the table uses double quotes `kodiak: merge.method = "merge"`

which one is correct?

Reply:
2nd though: how to use  `kodiak: merge.method = "rebase"` as a merge label in the kodiak.toml?

I guess this will not work?

```
automerge_label = ["automerge", "kodiak: merge.method = "merge""]
```

Reply:
Yeah, I think your example of

```
automerge_label = ["automerge", "kodiak: merge.method = 'merge'"]
```

is what you need.

I'm not sure how TOML handles escaping quotes.

I've make a PR to update the docs: https://github.com/chdsbd/kodiak/pull/533